Meetings/2022-02-16=meeting-minutes.md

```diff
@@ -0,0 +1,72 @@
+# Maintainers' Meeting 2022-02-16
+
+## Meeting Attendees
+
+- Rob Prouse - @rprouse
+- Mattias Karlsson - @devlead
+- Andrew White - @AndrewTriesToCode
+- Dylan Perks - @Perksey
+- Glenn Watson - @glennawatson
+- James Newton-King - @JamesNK
+- James Turner - @Turnerj
+- Michael Hawker
+- Pascal Berger - @pascalberger
+- Robin Krom
+- Tanner Gooding - @tannergooding
+
+## Recording
+
+**TBD**
+
+### Virtual Inc Staff Observers
+
+- Nadeem Abutaa
+- Tom Pappas
+- Nicole Miller
+
+## Slides
+
+## Agenda
+
+- Review [previous meeting minutes](2022-01-12-meeting-minutes.md)
+- Leaving the .NET Foundation process
+  - Walkthrough the [latest draft](https://github.com/dotnet-foundation/Home/discussions/68#discussioncomment-2168432).
+  - Feedback / Q & A
+  - Road map
+- Signing owner lockout on nuget
+- Centralizing and providing guidance to project maintainers for common questions/issues that arise and how to address them from [this thread here](https://github.com/dotnet-foundation/Home/discussions/61).
+- Other business
+
+## Notes
+
+- Reviewed previous meeting minutes
+- Introduction of new member Robin Krom
+- Discussed the process to leave the foundation,
+  - Tanner Gooding, it would be good to work with the NuGet team to find process for switching to unsigned packages going forward
+  - How do we deal with it when the original maintainers are no longer available?
```

Comment:
There was discussion in the meeting that your legal department indicated that even if a previous maintainer passed away you'd have to deal with the estate. People may not be available due to backpacking or other travel which is also another consideration.